docs/source/use_cases/privacy_protection_faces.md

```diff
@@ -0,0 +1,61 @@
+# Privacy Protection (Faces)
+
+## Overview
+
+As organisations collect more data, there is a need to better protect the identities of individuals in public and private places. AI Singapore has developed a solution that performs face anonymisation. This can be used to comply with the General Data Protection Regulation (GDPR) or other data privacy laws.
```

Comment:
This is a nice intro 👍

Reply:
We'll need to include a GIF

Reply:
Additionally, the GIFs should be added to GIF table in README.md and https://github.com/aimakerspace/PeekingDuck/blob/dev/docs/source/index.md. Perhaps the 2 privacy ones are side by side, and the mask detection one below it?